src/Components/Cards/FavoriteCard.scss

```diff
@@ -0,0 +1,9 @@
+.favorite-card-image {
+  height: 45%;
+  width: 35%;
+}
+
+.favorite-card-container {
+  display: flex;
+  flex-direction: row;
```

Comment:
I don't know the exact application here, but just something to take a look at. I've found my past projects since row is the default value, it's not usually necessary to apply it unless you've double flexed your container. If that's the case, it might be a matter of either removing the display flex on line 7 or reconfiguring other places it's applied.
Oooooooorrrrrr........I could be totally wrong, in which case you should ignore this comment :)

Reply:
Thank you!!